Add files via upload

AnsKaz-yu opened the pull request.

AnsKaz-yu committed:
Add files via upload

anthark reviewed (changes requested):
Тут, наверное, всё в порядке, но... Я хочу проверить точно при помощи RenderDoc, а не из-под студии не запускается =( 

И та же проблема, что и в предыдущей работе - не все объекты удаляются при выходе.

AnsKaz-yu committed:
Add files via upload

AnsKaz-yu commented:
Я решила проблему с освобождением ресурсов, но не совсем понимаю, что делать с тем, что оно не запускается в RenderDoc. Могу предположить, что проблема в том, что я использовала загрузку directxtk_uwp через NuGet в VisualStudio, чтобы использовать WICTextureLoader. И, возможно, это и не даёт собрать проект в RenderDoc и нужно тогда как-то по-другому подключать библиотеку...

AnsKaz-yu committed:
Теперь должно запускаться вне VisualStudio

anthark reviewed (changes requested):
Всё равно не работает. Проблема с путями до ресурсов.

AnsKaz-yu committed:
Add files via upload

AnsKaz-yu commented:
Третья попытка... Увидела, что не до конца убрала зависимость от NuGet, и скорее всего автоматически искались те зависимости, а не напрямую загруженные в проект

anthark reviewed (changes requested):
Опять не получилось =( Шейдеры не находятся, поскольку рабочий путь другой при запуске не из-под студии

AnsKaz-yu committed:
Delete Lab5 directory

AnsKaz-yu committed:
Add files via upload

AnsKaz-yu commented:
Добавила ручную настройку рабочей директории. Надеюсь, в этот раз сработает)

anthark reviewed (changes requested):
Всё равно не работает =( Не, я могу, конечно, указать путь к файлу проекта .vcxproj в RendrDoc и тогда будет работать, но условие сдачи задачи, озвученное на первой же лекции было вполне четким - из-под студии и по клику на исполняемый файл

AnsKaz-yu committed:
Попытка номер n (надеюсь, последняя)

Я полностью пересобрала проект с нуля, загрузив туда старый код ( его проверила на зависимости и тд). Надеюсь, поможет

AnsKaz-yu committed:
Delete Lab5.sln

AnsKaz-yu committed:
Delete Lab5 directory

AnsKaz-yu committed:
Попытка номер n (надеюсь, последняя)

AnsKaz-yu committed:
Add files via upload

anthark reviewed (approved):
ОК, наконец-то действительно получилось =)
